lib/server/services/qualityControlFlag/QcFlagTypeService.js

```diff
@@ -180,6 +176,14 @@ class QcFlagTypeService {
         }
 
         if (filter) {
+            /**
+             * @param {number[]} [options.filter.ids] quality control flag ids to filter with
+             * @param {boolean} [options.filter.bad] quality control flag boolean to filter bad or not-bad QC flags
+             * @param {boolean} [options.filter.archived] quality control flag boolean to filter archived or not-archived QC flags
+             * @param {string[]} [options.filter.names] quality control flag type name to filter with, single token is treated as pattern
+             * @param {string[]} [options.filter.methods] quality control flag type method to filter with, single token is treated as pattern
```

Comment:
Same here, not sure the param applies for an object, it might be `@type` instead